### DISCUSSION TITLE
How to force Azuracast to update metadata from file?

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [AzuraCast/AzuraCast#7466] [Q&A] [answered]
Is there a way to force Azuracast to get the metadata from file? I see Azuracast keeps "old" metadata in my syndicated files. This leads to "cut off" tracks in combination with Autocue.

Here is an example what Azuracast has stored
![Screenshot_20240924_152903](https://github.com/user-attachments/assets/649e395c-9ee0-4824-860a-9d72d7bc38fb)
But this is what it should be
![Screenshot_20240924_152945](https://github.com/user-attachments/assets/c3366fe4-bc9c-4c71-a9bb-8bb4de00c873)

**Comment by mishcazzulani** [comment]
You answered my question about the same topic a couple of days ago! See post #7449 

It would be great to have a checkbox  in the media editor or an api call to force azuracast to reprocess the media, as with my workaround the duration field is not updated.

**Reply by LordHelmchen666** [reply]
Yes. If you use liq_cue_file:true then Autocue ignores Azuracast values and newly calculates all things. But i don't want that. It makes no sense to calculate on download and then recalculate on playout. I would like Azuracast to pick up these new values from file. At the moment it is only possible to send via API as you wrote to avoid that.

**Comment by BusterNeece** [comment, marked as answer]
Newer Rolling releases no longer cache the autocue calculated data in the AzuraCast custom media editor (as it's prone to changing and this wasn't a reliable solution), but you can always reprocess the metadata or clear extra metadata via the corresponding options on the Media page's "More" dropdown.

**Reply by LordHelmchen666** [reply]
Thanks Buster. It is more logical that AzuraCast don't cache Autocue results. That way it is always possible to overwrite Autocue results if needed.
I already tried "Reprocess" but AzuraCast did not update Metadata from file. Even after waiting half an hour it still had old values.
So for now I delete all extra Metadata from AzuraCast. And observe how it works.
